modules/cluster-logging-feature-reference.adoc

```diff
@@ -0,0 +1,116 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: REFERENCE
+[id="logging-feature-ref_{context}"]
+= Collector features
+
+.Log Sources
+[options="header"]
+|===============================================================
+| Feature                   | Fluentd  | Vector
+| App container logs        | &#10003; | &#10003;
+| App specific routing      | &#10003; | &#10003;
+|...by namespace            | &#10003; | &#10003;
+| Infra container logs      | &#10003; | &#10003;
+| Infra journal logs        | &#10003; | &#10003;
+| Kube API audit logs       | &#10003; | &#10003;
+| Openshift API audit logs  | &#10003; | &#10003;
+| Open Virtual Network (OVN) audit logs| &#10003; | &#10003;
+|===============================================================
+
+.Outputs
+[options="header"]
+|==========================================================
+| Feature              | Fluentd  | Vector
+| Elasticsearch v5-v7  | &#10003; | &#10003;
+| Fluent forward       | &#10003; |
+| Syslog RFC3164       | &#10003; |
+| Syslog RFC5424       | &#10003; |
+| Kafka                | &#10003; | &#10003;
+| Cloudwatch           | &#10003; | &#10003;
+| Loki                 | &#10003; | &#10003;
+|==========================================================
+
+.Authorization and Authentication
+[options="header"]
+|=================================================================
+| Feature                     | Fluentd  | Vector
+| Elasticsearch certificates  | &#10003; | &#10003;
+| Elasticsearch username / password | &#10003; | &#10003;
+| Cloudwatch keys             | &#10003; | &#10003;
+| Cloudwatch STS              | &#10003; |
+| Kafka certificates          | &#10003; | &#10003;
+| Kafka username / password   | &#10003; | &#10003;
+| Kafka SASL                  | &#10003; | &#10003;
+| Loki bearer token           | &#10003; | &#10003;
+|=================================================================
+
+
+.Normalizations and Transformations
+[options="header"]
+|============================================================================
+| Feature                                | Fluentd  | Vector
+| Viaq data model - app                  | &#10003; | &#10003;
+| Viaq data model - infra                | &#10003; | &#10003;
+| Viaq data model - infra(journal)       | &#10003; | &#10003;
+| Viaq data model - audit linux          | &#10003; | &#10003;
+| Viaq data model - audit kube api       | &#10003; | &#10003;
+| Viaq data model - audit openshift api  | &#10003; | &#10003;
+| Viaq data model - ovn                  | &#10003; | &#10003;
+| Loglevel Normalization                 | &#10003; | &#10003;
+| JSON parsing                           | &#10003; | &#10003;
+| Structured Index                       | &#10003; | &#10003;
+| Multiline error detection              | &#10003; | &#10003;
```

Comment:
Multiline error detection story is added to GA phase 2 which needs to be completed. https://issues.redhat.com/browse/LOG-1796